anvil/core/Cargo.toml

```diff
@@ -11,6 +11,8 @@ foundry-evm = { path = "../../evm" }
 ethers-core = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
+bytes = { version = "1.1.0" }
```

Comment:
not sure this matters since this is only used in other crates, and ethers currently also uses `1.1.0`, but I'd like to set it to `1.1` here

Reply:
sounds good, changed to "1.1"